src/foraging_gui/Dialogs.py

```diff
@@ -1294,11 +1292,9 @@ def _OpenSaveFolder(self):
                 subprocess.Popen(['explorer', os.path.join(os.path.dirname(os.path.dirname(self.MainWindow.Ot_log_folder)),'behavior-videos')])
             except Exception as e:
                 logging.error(str(e))
-                self.WarningLabelOpenSave.setText('No logging folder found!')
-                self.WarningLabelOpenSave.setStyleSheet(self.MainWindow.default_warning_color)
+                logging.warning('No logging folder found!', extra={'tags': self.MainWindow.warning_log_tag})
         else:
-            self.WarningLabelOpenSave.setText('No logging folder found!')
-            self.WarningLabelOpenSave.setStyleSheet(self.MainWindow.default_warning_color)
```

Comment:
![image](https://github.com/user-attachments/assets/d9670386-d83b-44a5-b463-6db3ad079b45)

Reply:
I would probably duplicate this one, since its relevant to the camera dialog